core/src/main/resources/AppConstants.properties

```diff
@@ -105,6 +105,8 @@ browse.messages.max = 100
 browse.messages.order = DESC
 browse.errors.order = ASC
 
+transactionlog.dir=${log.dir}/${instance.name.lc}
```

Comment:
Misschien goed om hier ``${log.dir}/transactionlogs/${instance.name.lc}`` van te maken, zodat je, als je die naam ziet, er nog even over na kan denken of je ze weg wilt gooien.